src/amr/resources_manager/amr_utils.h

```diff
@@ -179,13 +180,13 @@ namespace amr
                 dl[iDim]     = pDl[iDim];
             }
         }
-        else
+        else // temporary patch
```

Comment:
could you be a little more verbos in the comment?
just this, we'll forget pretty soon what it means...
needs to explain that temp patch created while running in parallel and it does not have a geometry (hence ptr is nullptr) and thus we hard-code a dumb layout... but since we think it's only for temp patches and only samrai uses them, and it does not know about our layout, that's probably ok....

thx

Reply:
sure and will link the samrai issue you made

Reply:
![image](https://user-images.githubusercontent.com/30491/93351568-2f163e80-f83a-11ea-8706-2cbadff08229.png)

Reply:
done

Reply:
https://github.com/Dekken/PHARE/blob/autodump/src/amr/resources_manager/amr_utils.h#L183